src/com/rcv/Tabulator.java

```diff
@@ -847,20 +848,56 @@ void doAudit(List<CastVoteRecord> castVoteRecords) {
 
   // vote transfer rule to use in multi-seat elections
   enum MultiSeatTransferRule {
-    TRANSFER_FRACTIONAL_SURPLUS,
-    TRANSFER_WHOLE_SURPLUS,
-    TRANSFER_RULE_UNKNOWN,
+    TRANSFER_FRACTIONAL_SURPLUS("transferFractionalSurplus"),
+    TRANSFER_WHOLE_SURPLUS("transferWholeSurplus"),
+    TRANSFER_RULE_UNKNOWN("transferRuleUnknown");
+
+    private final String label;
+
+    MultiSeatTransferRule(String label) {
+      this.label = label;
+    }
+
+    static MultiSeatTransferRule getByLabel(String labelLookup) {
```

Comment:
Darn. When I saw that you were using this above, I was hoping that it was a built-in thing that we had overlooked. But anyway, yeah, this is better encapsulation.